benchmark/worker/benchmark_client.go

```diff
@@ -351,6 +357,11 @@ func (bc *benchmarkClient) doCloseLoopStreaming(conns []*grpc.ClientConn, rpcCou
 func (bc *benchmarkClient) getStats(reset bool) *testpb.ClientStats {
 	var timeElapsed float64
 	mergedHistogram := stats.NewHistogram(bc.histogramOptions)
+	timeElapsed = time.Since(bc.lastResetTime).Seconds()
```

Comment:
sorry I'm not sure if I understand here. 

Of the time elapsed since the last reset, I was neglecting the time taken to merge histograms. Do you mean to add the keep sampling the `timeElapsed` after merging histograms?